socialdistribution/views.py

```diff
@@ -1,9 +1,25 @@
 from django.urls import reverse_lazy
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import redirect, render
+from django.views.generic.list import ListView
+
+from posts.models import Post
 
 
 def root(request: HttpRequest) -> HttpResponse:
     if request.user.is_anonymous:
         return redirect(reverse_lazy('auth_provider:login'))
-    return render(request, 'stream.html')
+    return redirect('/stream')
```

Comment:
Might be better to use named urls (e.g. `reverse('stream')`)

Reply:
Great idea, didn't know this could be done